Harvest.Web/Controllers/RateController.cs

```diff
@@ -160,16 +160,47 @@ public async Task<ActionResult> Edit(int id, RateEditModel model)
 
             var user = await _userService.GetCurrentUser();
 
+            //TODO: When the rate is actually used, check the db to see if we need to archive. (If I just created it, and edit the rate, we don't need to archive)
+            var archive = rateToEdit.Price != model.Rate.Price;

```

Comment:
@jSylvestre actually I think while tracking change of rate itself is probably useful, the real reason to copy is when the account changes.  I'm on the fence about what should happen if they just change the billing unit or description -- but for now the biggest thing would be if the account changes we need to archive.